### ISSUE TITLE
Non interactive mode for "oci session authenticate"

### ISSUE THREAD

**Issue by robert-heinzmann-logmein** [oracle/oci-cli#438]
After an upgrade to 3.0 of the cli the call for oci session authenticate is now always interactive

```
export OCI_CLI_CONFIG_FILE=myociprofile
export OCI_CLI_PROFILE=DEFAULT
export OCI_CLI_AUTH=security_token
oci session authenticate --tenancy-name $TENANT --region $HOME_REGION --profile-name $OCI_CLI_PROFILE --config-location $OCI_CLI_CONFIG_FILE
...
Enter a passphrase for your private key (empty for no passphrase):
...
```

It seems there is no way to make this **non interactive** (e.g. passing the passphrase as environment variable OR adding a cli option to not demand one).

**Comment by harshkumar-dev** [comment]
Hi Robert,
We will fix this in our next release. Please consider downgrading CLI to 2.26.4 version if you don't need any new feature released in 3.0.0. If this will cause significant difficulties for you in the mean time, please reach out to our product manager (zack.menegakis@oracle.com) and provide more details.

**Comment by vish1** [comment]
@robert-heinzmann-logmein  This issue has been fixed in the latest CLI release 3.0.1. Please see [link](https://github.com/oracle/oci-cli/blob/master/CHANGELOG.rst#301---2021-08-17) for changelog.
